coordinator/schedulepod/Dockerfile

```diff
@@ -0,0 +1,33 @@
+# RUN docker build -f ./metadata/Dockerfile . -t metadata from /serving
+# docker tag metadata:latest featureformcom/metadata:latest
+# docker push featureformcom/metadata:latest
+FROM golang:1.17-alpine
+
+WORKDIR /app
+
+COPY go.mod ./
+COPY go.sum ./
+
+COPY ./metadata/proto/metadata.proto ./metadata/proto/metadata.proto
+RUN apk update && apk add protobuf-dev && go install google.golang.org/grpc/cmd/protoc-gen-go-grpc@latest && go install google.golang.org/protobuf/cmd/protoc-gen-go@latest
+ENV PATH /go/bin:$PATH
+RUN protoc --go_out=. --go_opt=paths=source_relative --go-grpc_out=. --go-grpc_opt=paths=source_relative ./metadata/proto/metadata.proto
+
+COPY ./metadata/*.go ./metadata/
+COPY ./metadata/proto/ ./metadata/proto/
+COPY ./metadata/search/ ./metadata/search/
+COPY ./metadata/server/server.go ./metadata/main/server.go
+COPY ./coordinator/*.go ./coordinator/
+COPY ./provider/ ./provider/
+COPY ./metadata/ ./metadata/
+COPY ./runner/ ./runner/
+COPY ./coordinator/schedulepod/*.go ./coordinator/schedulepod/
```

Comment:
Rename this directory to "tests" or something so we know its not part of the main project

Reply:
ok

Reply:
changed commit e26850c

Reply:
Change the parent directory of this to scheduling_tests. schedulepod looks like its part of the main coordinator logic when it should be clear its just for testing